src/config/language.c

```diff
@@ -150,53 +149,83 @@ static void CONFIG_ReadLangV1(FILE* fh)
             lookup0->pos = pos;
 
             lookup++;
-            strlcpy(strings + pos, line, len);
+            strlcpy(strings + pos, tempstring, len);
             pos += len;
         }
     }
     table_size = lookup - lookupmap;
 }
 #endif
 
-static void CONFIG_ReadLangV2(FILE* fh)
+static void ReadLangV2(FILE* fh)
 {
     u16 hash;
     unsigned pos = 0;
-    char line[MAX_LINE];
+    char tempstring[MAX_LINE];
     struct str_map *lookup = lookupmap;
```

Comment:
why is this here?  either use global tempstring or define a different variable, but please don't define one named 'tempsting'

Reply:
mismerge. want to use global tempstring to reduce stack usage.